codbex-payment-invoice-ext/generate/SalesInvoicePayment/api/GenerateSalesInvoicePaymentService.ts

```diff
@@ -0,0 +1,70 @@
+import { SalesInvoiceRepository as SalesInvoiceDao } from "../../../../codbex-invoices/gen/codbex-invoices/dao/salesinvoice/SalesInvoiceRepository";
```

Comment:
You could try to use absolute imports instead of relative:

```ts
import { SalesInvoiceRepository as SalesInvoiceDao } from "codbex-invoices/gen/codbex-invoices/dao/salesinvoice/SalesInvoiceRepository";
```

Reply:
I understood and fixed that